hip/static/styles/pages/poster_list_page.scss

```diff
@@ -3,5 +3,5 @@
   @include lists;
   @include a-tags;
 
-  padding: 0.75rem 2rem;
+  padding: 0.75rem 0.25rem;
```

Comment:
It looks like this change only affects the padding on the main column, rather than on each poster card. It looks to me like it's the `card-content` padding that's causing the issue on this page

Reply:
Got it, will fix.

Reply:
done